src/components/themes/purple-haze/theme.css

```diff
@@ -550,7 +545,7 @@ a[data-role=button] {
 }
 
 ::-webkit-scrollbar-track {
-    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, .3)
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, .3)
 }
```

Comment:
I'm wondering if maybe these rules are actually still necessary, I wouldn't think so.

Reply:
We should try to be as conservative as possible, I think. We can look into removing these eventually after the prefix stuff is taken care of.

I'm mainly trying to not make stuff break and have the change be as transparent as possible in the frontend :p